fmriprep/run_workflow.py

```diff
@@ -76,6 +76,11 @@ def main():
         'skull_strip_ants': opts.skull_strip_ants,
         'output_dir': op.abspath(opts.output_dir),
     }
+    settings['work_dir'] = settings['output_dir'] # other wfs assume this is set
```

Comment:
This does not make any sense. We have now an orphaned option.

Additionally, settings['work_dir'] is necessary along the process. We need to roll this back.